client/packages/electron/src/electron.ts

```diff
@@ -10,6 +10,9 @@ import {
 import HID from 'node-hid';
 import ElectronStore from 'electron-store';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path');
```

Comment:
```suggestion
import path from 'path';
```

This seems to work here...